terraform/modules/eks/serviceacc.tf

```diff
@@ -1,45 +1,54 @@
+
+# Wait for EKS cluster to be fully ready
 resource "time_sleep" "wait_for_kubernetes" {
   depends_on      = [aws_eks_cluster.ce7_grp_2_eks]
-  create_duration = "20s"
+  create_duration = "20s" # Pause to ensure cluster is stable
 }
 
+# Create namespace to organize our applications
 resource "kubernetes_namespace" "apps" {
```

Comment:
I manually created three namespaces: dev, uat, prod using kubectl.
But I think it's better to create those namespaces here in terraform.
Can help add those?